### PR TITLE
Add tokens task

### LINKED ISSUE #1971: Add dummy deployment with token addresses
It'd be handy to have a 'fake' deployment task we could reference to get the address of BAL and WETH, since we'll need this in multiple places. 

This would just be the outputs, with no inputs, index, etc., meaning some of the automatic checks will fail. Maybe we can have a task prefix that is ignored by `check-deployments` and friends? e.g. `00000-tokens`.

### PR DESCRIPTION
**Pull request by nventuro**
# Description

This is just so that we can reference these in other tasks. Many tasks already use these addresses (namely the Vault and ChildChainGaugeFactory), but I'd change those inputs later.

## Type of change

- [ ] Bug fix 
- [X] New feature 
- [ ] Breaking change 
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## Checklist:

- [ ] The diff is legible and has no extraneous changes
- [ ] Complex code has been commented, including external interfaces
- [ ] Tests are included for all code paths
- [ ] The base branch is either `master`, or there's a description of how to merge

## Issue Resolution

Closes #1971

### CODE REVIEW

**Comment by nventuro** [comment]
I changed my mind - let's use the inputs.

File: pkg/deployments/tasks/00000000-tokens/output/gnosis.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "BAL": "0x7eF541E2a22058048904fE5744f9c7E4C57AF717",
+  "WETH": "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d"
```
Comment (TomAFrench) [inline pkg/deployments/tasks/00000000-tokens/output/gnosis.json:3]: Let's add a readme to  this task explaining that Gnosis uses xDAI as it's native asset so "WETH" is really wrapped DAI. Similarly for the other chains which have their own native asset.

File: pkg/deployments/tasks/20220413-child-chain-gauge-factory/input.ts [hunk 2]
```diff
@@ -8,24 +8,21 @@ export type ChildChainLiquidityGaugeFactoryDeployment = {
 
 const AuthorizerAdaptor = new Task('20220325-authorizer-adaptor', TaskMode.READ_ONLY);
 const Vault = new Task('20210418-vault', TaskMode.READ_ONLY);
-const TestBalancerToken = new Task('20220325-test-balancer-token', TaskMode.READ_ONLY);
+const Tokens = new Task('00000000-tokens', TaskMode.READ_ONLY);
 
 export default {
   AuthorizerAdaptor,
   Vault,
```
Comment (TomAFrench) [inline pkg/deployments/tasks/20220413-child-chain-gauge-factory/input.ts:15]:
```suggestion
const BAL = new Task('00000000-tokens', TaskMode.READ_ONLY);

export default {
  AuthorizerAdaptor,
  Vault,
  BAL
```

This avoids us having to break out the different network configs.

File: pkg/deployments/tasks/20210418-vault/input.ts [hunk 3]
```diff
@@ -9,54 +9,32 @@ export type VaultDeployment = {
 };
 
 const Authorizer = new Task('20210418-authorizer', TaskMode.READ_ONLY);
+const Tokens = new Task('00000000-tokens', TaskMode.READ_ONLY);
```
Comment (TomAFrench) [inline pkg/deployments/tasks/20210418-vault/input.ts:12]:
```suggestion
const WETH = new Task('00000000-tokens', TaskMode.READ_ONLY);
```

Similarly to above.

File: pkg/deployments/tasks/00000000-tokens/output/gnosis.json (hunk 1 above)
Comment (nventuro) [reply pkg/deployments/tasks/00000000-tokens/output/gnosis.json:3]: These are jsons :( No docs there

Comment (nventuro) [reply pkg/deployments/tasks/00000000-tokens/output/gnosis.json:3]: I added a note in the readme.

**Review by TomAFrench — APPROVED** [review]

**Pull request merged**